src/components/Table/TableToolbar/_table-toolbar.scss

```diff
@@ -26,6 +20,11 @@ div.#{$prefix}--toolbar-action.#{$prefix}--toolbar-search-container-expandable {
   padding-top: $spacing-01;
 }
 
+// small fix to allow tooltips to show outside the toolbar if the batch action isn't active
```

Comment:
Very nice, I was just fighting the clip-path myself, trying to figure out how to work around it without causing a breaking change. So you are sure it was only needed for the batch actions and not for anything else? What about custom toolbar content that might rely on it?

Reply:
I believe so @bjornalm I almost opened a bug on carbon to only set the clip-path initially if the batch actions was active to make sure they approved.